src/extension/module.js

```diff
@@ -4162,7 +4168,7 @@ import sampleRUM from './rum.js';
       window.hlx.sidekick.show();
     } else {
       // toggle sidekick
-      window.hlx.sidekick.toggle();
+      // window.hlx.sidekick.toggle();
```

Comment:
Need to discuss this. Seems this toggle was preventing the sidekick from opening after removing `nocache` from the query params.

Reply:
Is this being trigered twice if a `nocache` query param is present, because of the `history.replaceState()`?
I think removing this line will disable toggling by clicking the extension icon in all other cases...

Reply:
It's not being triggered twice.

From what I could tell this did not break toggling by clicking the extension icon. But maybe I'm not trying it in the right scenario. Probably good if you try it out when you are back.